pkg/cli/login/loginoptions.go

```diff
@@ -512,3 +618,39 @@ func (o *LoginOptions) serverProvided() bool {
 func (o *LoginOptions) tokenProvided() bool {
 	return len(o.Token) > 0
 }
+
+func (o *LoginOptions) getIssuerUrlForOIDC() (*oauthdiscovery.OauthAuthorizationServerMetadata, error) {
```

Comment:
https://github.com/coreos/go-oidc/blob/v3/oidc/oidc.go#L202

Reply:
ah, it is nice to use that instead of copying. Thanks.

Reply:
After thinking on this, I'd not prefer importing a new go module just for one helper function(more CVEs filed by scanners) unless I have to.

Instead I think, we can wrap this https://github.com/openshift/library-go/blob/cfcf2b4fbc878cad84f0552dfdabe1642b985f23/pkg/oauth/tokenrequest/request_token.go#L161-L200 to a helper function and oc can consume it seems like a better idea. WDYT?.